packages/redux/src/bags/actionTypes.ts

```diff
@@ -14,6 +14,22 @@ export const ADD_BAG_ITEM_REQUEST =
 export const ADD_BAG_ITEM_SUCCESS =
   '@farfetch/blackout-redux/ADD_BAG_ITEM_SUCCESS';
 
+/**
+ * Action type dispatched when the set promocodes from bag request fails.
+ */
+export const SET_BAG_PROMOCODES_FAILURE =
+  '@farfetch/blackout-core/SET_BAG_PROMOCODES_FAILURE';
```

Comment:
```suggestion
  '@farfetch/blackout-redux/SET_BAG_PROMOCODES_FAILURE';
```
Same for the remaining actions.

Reply:
Fixed 👍